spec/schemas/_common.yaml

```diff
@@ -2656,6 +2656,9 @@ components:
           format: int32
         index:
           $ref: '#/components/schemas/IndexName'
+        searchOnly:
```

Comment:
@mch2 This is unfortunately too late to change for the 3.0 release, but we should introduce `search_replica` as the replacement here and mark this field as deprecated.